app-connector-client/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@varkes/app-connector-client",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
All of these should be updated to 0.3.0 with the lerna version command

Reply:
done